jupyter-js-widgets/examples/web/test/index.js

```diff
@@ -11,23 +11,21 @@ const widgetsRendered = new Promise((resolve, reject) => {
 describe('index.html', function() {
     this.timeout(10000);
 
-    beforeEach(() => {
-        return widgetsRendered;
-    });
+    beforeEach(function() { return widgetsRendered; });
 
-    describe('textArea', () => {
-        it('exists', () => {
+    describe('textArea', function() {
+        it('exists', function() {
             expect(document.querySelector('textarea')).to.be.ok;
         });
-        it('correct value', () => {
+        it('correct value', function() {
             expect(document.querySelector('textarea').value).to.equal('test <b>text</b>');
         });
     });
-    describe('html', () => {
-        it('exists', () => {
+    describe('html', function() {
+        it('exists', function() {
             expect(document.querySelector('div.widget-html')).to.be.ok;
         });
-        it('correct value', () => {
+        it('correct value', function() {
```

Comment:
I think that this file was ES6 already, so we probably don't need to change that.